packages/sdk-demo/src/components/Layout/index.tsx

```diff
@@ -73,12 +75,21 @@ export const DefaultLayout = ({ children, siderProps }: DefaultLayoutProps) => {
           <Menu />
           <Box
             sx={{
+              width: '100%',
               position: 'absolute',
               bottom: 0,
-              mb: 2,
+              p: 2,
             }}
           >
-            {siderProps?.footer}
+            <Stack direction="column" spacing={2} ml={2}>
+              {/*Themes are unfinished.*/}
+              {/*We want to show the theme switcher only in development mode and on the dev deployment only.*/}
+              {(process.env.NODE_ENV === 'development' ||
+                location.pathname.indexOf('dev.monite.') > 0) && (
```

Comment:
You probably where thinking of `location.hostname`, and its usage with 
`location.hostname.indexOf('dev.monite.')`.
Btw, it is always better to check, is the `location` really exists (the code could run not only on in the browser): `typeof location !== 'unefined'`

Reply:
It would be better to use `api_url` from `/config.json` to check stand instead of locale.
Also, we can add a `stand` to `config.json` to avoid parsing strings.

Reply:
fixed